contents/javascript/Javascript-promises.md

```diff
@@ -90,57 +86,118 @@ These situations correspond to the 3 states of Promises in Javascript.
 Promises provide the ability to specify how the execution of some part of your code would depend on the status of an asynchronous operation. It can now wait for the asynchronous operation to resolve first before doing any work on its result.
 
 ## Quickstart
+Here, we will be showing you some sample code to get started with promises. We understand there is some style of programming you are used to. So here, we are giving you a few examples, first using callbacks (which you are more familiar with), then switching over to using promises.
```

Comment:
`We understand there is some style of programming you are used to.`: not sure why we have this sentence here. You can simplify as,
>Let's learn how to go from callbacks to promises using a few examples.